Casks/p/prismlauncher.rb

```diff
@@ -22,6 +22,7 @@
   depends_on macos: ">= :high_sierra"
 
   app "Prism Launcher.app"
+  binary "#{appdir}/Prism Launcher.app/Contents/MacOS/prismlauncher", target: "prismlauncher"
```

Comment:
```suggestion
  binary "#{appdir}/Prism Launcher.app/Contents/MacOS/prismlauncher"
```

The `target` is not required, since it does not differ from the defaults.